db_schema/readrsql.patch.6.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE comments ( id BIGINT UNSIGNED PRIMARY KEY AUTO_INCREMENT, body varchar(500), og_title varchar(256), og_description varchar(256), og_image varchar(128), like_amount int, parent_id bigint UNSIGNED, resource varchar(128), created_at datetime, updated_at datetime, author BIGINT UNSIGNED, active tinyint, status tinyint, ip int unsigned);
+
+CREATE TABLE comments_reported ( id BIGINT UNSIGNED PRIMARY KEY AUTO_INCREMENT, comment_id BIGINT UNSIGNED, reporter BIGINT UNSIGNED, reason varchar(500), solved tinyint, created_at datetime, updated_at datetime);
```

Comment:
need ip here.

Reply:
Do we also need IP column for comments_reported table?

Reply:
yes.